README.md

```diff
@@ -1,27 +1,32 @@
 # NServiceBus.RavenDB
 
-The official [NServiceBus](https://github.com/Particular/NServiceBus) persistence implementation for [RavenDB.](https://ravendb.net/)
+NServiceBus.RavenDB is the official NServiceBus persistence implementation for [RavenDB](https://ravendb.net/).
```

Comment:
```suggestion
NServiceBus.RavenDB is the official [NServiceBus](https://github.com/Particular/NServiceBus) persistence implementation for [RavenDB](https://ravendb.net/).
```

To remain consistent with others like https://github.com/Particular/NServiceBus.Gateway.RavenDB/pull/517/files, have the NServicebus link to the repo

Reply:
Alas, I wasn't consistent with this 😬 But I don't mind having it for sure.a